.test-infra/jenkins/PostcommitJobBuilder.groovy

```diff
@@ -42,7 +42,9 @@ class PostcommitJobBuilder {
                             jobDefinition = {}) {
     PostcommitJobBuilder jb = new PostcommitJobBuilder(scope, jobDefinition)
     jb.defineAutoPostCommitJob(nameBase)
-    jb.defineGhprbTriggeredJob(nameBase + "_PR", triggerPhrase, githubUiHint, false)
+    if (triggerPhrase) {
```

Comment:
Would you mind adding a javadoc comment that describes this new behavior ("pass a null/empty triggerPhrase to disable phrase triggering")

Reply:
Done. Added Javadoc for all the parameters

Reply:
Done, added javadoc for all the parameters.